docs/sources/datasources/phlare.md

```diff
@@ -0,0 +1,92 @@
+---
+aliases:
+- /docs/grafana/latest/datasources/phlare/
+- /docs/grafana/latest/features/datasources/phlare/
+  description: Horizontally-scalable, highly-available, multi-tenant continuous profiling aggregation system. OSS profiling solution from Grafana Labs.
+  keywords:
+- grafana
+- phlare
+- guide
+- profiling
+  title: Phlare
```

Comment:
```suggestion
  title: Phlare data source
```

Reply:
Actually did not change it after all. In https://grafana.com/docs/grafana/latest/datasources/ we show it on the left under Datasources header so it looks weird to repeat that after each name. We have that right now only with influx not sure if accidental or not.